docs/community_resources/release_process.md

```diff
@@ -0,0 +1,62 @@
+# eZ Platform release process and roadmap
+
+## Release process
+
+eZ Platform has two distributions:
+
+- eZ Platform is an open source Content Management System (CMS) developed by eZ Systems together with the open source community. eZ Platform's code base is available on GitHub under the GPLv2 license. eZ Platform comes with no commercial support and maintenance services. It is supported by the community on public channels.
+- eZ Platform Enterprise Edition (EE) is commercial software available under eZ Business User License (BUL) to eZ Enterprise subscribers. It is comprised of eZ Platform Open Source edition, additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+
+We manage the release of eZ Platform using an agile iterative process and a continuous software development model, which is why we provide two kinds of eZ Platform releases:
+
+- Long Term Support releases (LTS) which are supported by eZ Systems for a long period of time (see [support lifecycle below](#support-lifecycle)), for eZ Enterprise subscribers.
+- Fast Track releases (FT) give access to the latest features and are supported for a short period of time, only until the next FT release is introduced. These are supported for both the open source community and eZ Enterprise subscribers.
```

Comment:
`only until the next FT release is introduced` => `maintained only until the next FT release is introduced`

context: https://support.ez.no/Public/Service-Life  As in with exception of 2.2 they all have 3 months maintenance + 3 months support.